lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -357,7 +357,7 @@ defmodule Kernel.SpecialForms do
       iex> Kernel.'+'(1, 2)
       3
 
-  Note that `Kernel."HELLO"` will be treated as a remote call and not an alias.
+  Note that `Kernel."function_name"` will be treated as a remote call and not an alias.
```

Comment:
@lexmag that would be the case if we were using `MyModule`. However, since it is `Kernel`, I would stick with "FUNCTION_NAME" or change everything to `MyModule."MY_FUNCTION"`. I think the Kernel one is clearer, given the previous example is `Kernel.+`, so I would ship it as is. :heart: